task2/rest-api-basics/repository/src/main/java/com/epam/esm/repository/config/RepositoryConfig.java

```diff
@@ -27,7 +27,7 @@ public JdbcTemplate jdbcTemplate() {
         return jdbcTemplate;
     }
 
-    @Bean
+    @Bean(destroyMethod = "close")
```

Comment:
What is the reason for using destroyMethod = "close"?

Reply:
> What is the reason for using destroyMethod = "close"?

I thought if Hikari not autoclosable that put close on bean. Should I delete this?

Reply:
As I understand, close or shutdown methods will be automatically marked as destroyMethod.
![image](https://user-images.githubusercontent.com/24493000/105823311-0c591b00-5fce-11eb-90a1-4d00bbd95aac.png)

Reply:
> As I understand, close or shutdown methods will be automatically marked as destroyMethod.
> ![image](https://user-images.githubusercontent.com/24493000/105823311-0c591b00-5fce-11eb-90a1-4d00bbd95aac.png)

Thank you for your attention. I will fix it.